### PR TITLE
New CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ryansb**
Contributor docs are important, let's have some! This is a first pass at
it, and feedback (especially in the form of pull requests) is welcome.

@GabeIsman @thequbit @ismcodes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+We have a few tags to help us navigate the issues page:
+
+- "bug" - Bugs that need squashing because something isn't working right.
+- "coding" - Tasks that involve programming, either frontend or backend.
+- "design" - This means feedback is desired. When an issue has this tag, it's
+    often about a future feature or a change to the project.
+- "documentation" - All things documentation. Developer docs, user docs,
+    installation docs, you name it.
+- "duplicate" - Exactly what it says on the tin.
+- "enhancement" - New features, or improvements to existing ones.
```
Comment (ismcodes) [inline CONTRIBUTING.md:69]: Enhancement doesn't connote a new feature IMO. Maybe one tag for enhancing existing code, and then a "new feature" one? @ryansb

Comment (ryansb) [inline CONTRIBUTING.md:69]: We originally had both "feature" and "enhancement" and at the meeting I think we decided we could use one tag for both.

**Comment by thequbit** [comment]
This looks great @ryansb, thank you for putting it together.

@ismcodes what are your additional thoughts on feature vs. enhancement?  Are you okay with a single tag?  Our thought was to minimize the number of labels to keep things concise.  @ismcodes if you're all good, I'll merge this PR.

**Comment by ismcodes** [comment]
@thequbit On one hand, I would think that tags would be used to search for issues as specifically as possible. On the other, keeping things concise is good and there isn't much of a difference between adding a new feature and improving a current one. 
So yeah, I think keeping it like this is good.

**Comment by thequbit** [comment]
Looks great! Merging.

**Pull request merged**